packages/nodes-base/nodes/Hubspot/Hubspot.node.ts

```diff
@@ -1944,6 +2233,191 @@ export class Hubspot implements INodeType {
+									...(filter.value !== undefined ? { value: filter.value } : {}),
+									...(filter.highValue !== undefined ? { highValue: filter.highValue } : {}),
+									...(filter.values !== undefined ? { values: filter.values } : {}),
+									propertyName: filter.propertyName,
+									operator: filter.operator,
+								})),
+							}),
+							);
+
+							const body = {
+								...(additionalFields.properties ? { properties: additionalFields.properties } : {}),
+								...(bodyFilterGroups ? { filterGroups: bodyFilterGroups } : {}),
+								...(limit ? { limit } : {}),
+								...(additionalFields.query ? { query: additionalFields.query } : {}),
+								...(additionalFields.sortBy ? { sorts: [additionalFields.sortBy] } : {}),
+							};
+
+							const endpoint = `/crm/v3/objects/${customObjectType}/search`;
+
+							if (returnAll) {
+								responseData = await hubspotApiRequestAllItems.call(this, 'results', 'POST', endpoint, body);
+							} else {
+								const response = await hubspotApiRequest.call(this, 'POST', endpoint, body);
+								responseData = response.results;
+							}
+
+						}
+						if (operation === 'define') {
+							const name = this.getNodeParameter('name', i) as string;
+							const labels = this.getNodeParameter('labels', i) as IDataObject;
+							const properties = (this.getNodeParameter('properties.values', i, []) as IDataObject[]).map(property => {
+								if (!property.options) return property;
+								return {
+									name: property.name,
+									label: property.label,
+									...property.options as IDataObject,
+								};
+							});
+							const additionalFields = this.getNodeParameter('additionalFields', i) as IDataObject;
+							const primaryDisplayProperty = this.getNodeParameter('primaryDisplayProperty', i) as string;
+
+							const body: IDataObject = {
+								name,
+								primaryDisplayProperty,
+								...labels,
+								properties,
+							};
+
+							Object.keys(additionalFields).forEach(key => body[key] = (additionalFields[key] as string).split(','));
+
+							const endpoint = `/crm/v3/schemas`;
+							responseData = await hubspotApiRequest.call(this, 'POST', endpoint, body);
+						}
+						if (operation === 'delete') {
+							const idProperty = this.getNodeParameter('idProperty', i) as string;
+							const objectId = this.getNodeParameter('objectId', i) as string;
+
+							let realObjectId = objectId;
+
+							if (idProperty) {
+								const qs = {
+									idProperty,
+									properties: [idProperty],
+								};
+								const getEndpoint = `/crm/v3/objects/${customObjectType}/${objectId}`;
+								const idResponseData = await hubspotApiRequest.call(this, 'GET', getEndpoint, {}, qs);
+								realObjectId = idResponseData.id;
+							}
+
+							const endpoint = `/crm/v3/objects/${customObjectType}/${realObjectId}`;
+							responseData = await hubspotApiRequest.call(this, 'DELETE', endpoint);
```

Comment:
response is always undefined, in case if operation was successful we need to return {success: true}